tests/migration_test.go

```diff
@@ -539,6 +539,8 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		})
 		Context("with setting guest time", func() {
 			It("should set an updated time after a migration", func() {
+				tests.SkipMigrationTestIfRunnigOnKindInfra()
```

Comment:
this test fails before ipv6 migration PR on non ipv6 lanes
imo it shouldnt be the same skip used
it shouldnt block my PR or hitch hiked on it please

Reply:
The timeout increase was a mistake of mine, it shouldn't have been pushed (I've removed it already).

On other lanes, the test fails *after* migration: check for instance https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3249/pull-kubevirt-e2e-k8s-1.15-ceph/1255441838593019905. In it, it fails because the guest agent fails to update the guest's datetime **after** migration.

In the IPv6, it fails with the following error on the virt-handler pod, when a migration is requested - before migrating:
```
{
    "component":"virt-handler",
    "level":"error",
    "msg":"unable to create outbound leg of proxy to host fd00:10:244::4f:35943",
    "pos":"migration-proxy.go:384",
    "reason":"dial tcp: address fd00:10:244::4f:35943: too many colons in address",
    "timestamp":"2020-04-28T06:58:09.810905Z"
}
```

They **are** 2 different issues, and I'm only addressing one (the IPv6 lane). 

Furthermore, your PR fixes the IPv6 lane by making the migration-proxy handle IPv6 addresses. I do not understand why you say this 'blocks' your PR. Your PR is required to fix this.

Reply:
Hi, thanks for the explanation,

i mean that when i would be ready to push my PR , if possible that i wouldn't have to fix this test as well directly as a preliminary step, maybe this is the situation ? (since you said there are 2 different issues)

better to separate collision domain and make things independent,
else we are fighting again and again with prow and flakiness